### DISCUSSION TITLE
React Comunity

### DISCUSSION THREAD

**Discussion by mustafajhon** [community/community#45994] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

**Hey React Developers...**
### I am looking for those who wants to practice learning react through online practice.

**Comment by ettaboyle** [comment]
Welcome to the GitHub Community, [mustafajhon](https://github.com/mustafajhon), we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it to "General" from "Profile"for you. Good luck!
